win_setupgui/src/config.rs

```diff
@@ -45,10 +44,9 @@ unsafe extern "C" fn ConfigDSNW(
             }
             ODBC_CONFIG_DSN => match dsn_opts.from_private_profile_string() {
                 Ok(dsn) => config_dsn(dsn, request),
-                Err(e) => {
+                Err(_e) => {
                     // we've somehow attempted to read a value from the DSN
                     // that is longer than the registry allows (at the time of writing!)
```

Comment:
why was this dbg removed?

Reply:
I've removed the debug statements from this pr, knowing that logging statements are being proposed. I wanted to get rid of as many uses of `chrono` here as I could.

Reply:
If so, please add a `TODO - Log this` to not forget about it ;)